### DISCUSSION TITLE
Change opacity of Windows 10 style window switcher without acrylic or mica

### DISCUSSION THREAD

**Discussion by Poopooracoocoo** [valinet/ExplorerPatcher#298] [Ideas]
I often have battery saver on so mica and acrylic get turned off :( Opacity is a really important part of the alt+tab experience for me tho

**Comment by valinet** [comment]
At some point, it will be added in. Basically a correct implementation for a layered window has to be made, where the transparency is controlled by the opacity setting, and care should be taken to have the graphics drawn fully opaque. Idk, at some point I will take care of that as well, or maybe someone will step in and take care of it as well.

Thanks

**Comment by valinet** [comment]
I have added this in the latest pre-release version (40.2), along with many other improvements to this custom window switcher. You can now basically have it be semi-transparent when on battery saver/when the transparency effects are generally disabled on the system.
